sandbox/settings.py

```diff
@@ -290,6 +290,8 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
                 environ_name="EDX_COURSE_REGEX",
                 environ_prefix=None,
             ),
+            # Synchronization
+            "COURSE_RUN_SYNC_PROTECTED_FIELDS": [],
```

Comment:
I find the word "protected" confusing because, on the contrary, these are fields which we allow to update manually.
Maybe `COURSE_RUN_SYNC_NO_UPDATE_FIELDS`

Reply:
Good idea 🙏